tests/data/colors/js-perl.svg

```diff
@@ -342,80 +342,68 @@
 <text id="matched" x="1090.00" y="373" > </text>
 <g id="frames">
 <g >
-<title>Stub:JSEntryStub (2 samples, 100.00%)</title><rect x="10.0" y="133" width="1180.0" height="15.0" fill="rgb(88,200,200)" rx="2" ry="2" />
-<text  x="13.00" y="143.5" >Stub:JSEntryStub</text>
-</g>
-<g >
-<title>uv__async_io (2 samples, 100.00%)</title><rect x="10.0" y="245" width="1180.0" height="15.0" fill="rgb(223,84,84)" rx="2" ry="2" />
-<text  x="13.00" y="255.5" >uv__async_io</text>
+<title>uv__io_poll (2 samples, 100.00%)</title><rect x="10.0" y="261" width="1180.0" height="15.0" fill="rgb(223,84,84)" rx="2" ry="2" />
+<text  x="13.00" y="271.5" >uv__io_poll</text>
```

Comment:
I'm not sure I follow why this file had to change in this way as a result of this change?

Reply:
I actually don't know. Same command line, same output (at least visually...)

Reply:
That's pretty disturbing, because this changes locations, not just colors... Could just have been merging changes from `master` though I suppose.

Reply:
It actually only change the svg element layout... I thought it could be because I'm juggling between multiple `flamegraph.pl`, but some options (`--bgcolors`) are not available on the version we point at in the `inferno` repo, so I could not have use the wrong version.

Reply:
`flamegraph.pl` output does not seem stable ? I generated `js-perl.svg` again, and I have other diffs...

Reply:
Ohh, I wonder if this is because it uses a map to keep attributes, and iteration over the map is probably non-deterministic :/

Separately, feel free to do update the `flamegraph` submodule!